docker/entrypoints/dev.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ "$(psql -XtA -U postgres -h $DB_HOST funnel -c "select count(*) from information_schema.tables where table_schema = 'public';")" = "0" ]; then
+    flask dbcreate
+    flask db stamp
+else
+    flask db upgrade head
+fi
```

Comment:
I'm still uncomfortable with this. It should be explicit, not implicit, as it's risky to do this when in the middle of writing a migration.

Reply:
So, we just remove the else part. We can always run `docker compose exec -it dev flask db upgrade head` manually.